### PR TITLE
Monthly translation update for ASA (Aug '23)

### PR DESCRIPTION
**Pull request by eng-prod-CI-bot-okta**
ASA translation for the latest commit: https://github.com/okta/okta-help/commits/docs_translations_ASA

### CODE REVIEW

File: asa/ja-jp/content/topics/adv_server_access/docs/audit-events-with-okta-syslog.htm [hunk 1]
```diff
@@ -311,7 +311,7 @@ <h2>前提条件</h2>
                                                             <p><span class="mc-variable okta-variables.ProductName variable">Okta</span> <span class="mc-variable okta-feature-names.Administrator_dashboard variable"> Admin Console</span>にサインインしていることを確認します。</p>
                                                         </li>
                                                         <li>
-                                                            <p>必要な権限が<span class="mc-variable okta-variables.ProductName variable">Okta</span> org全体の設定にあることを確認します。詳細については、「標準的な管理者ロールと権限」を参照してください。</p>
+                                                            <p>必要な権限が<span class="mc-variable okta-variables.ProductName variable">Okta</span> org全体の設定にあることを確認します。詳細については、「<a href="https://help.okta.com/oie/ja-jp/Content/Topics/Security/administrators-admin-comparison.htm" target="_blank">標準的な管理者ロールと権限</a>」を参照してください。</p>
```
Comment (paulwallace-okta) [inline asa/ja-jp/content/topics/adv_server_access/docs/audit-events-with-okta-syslog.htm:314]: We have an absolute URL here. Making a note, will need to look at source.

File: asa/ja-jp/content/topics/adv_server_access/docs/audit-events-with-okta-syslog.htm [hunk 2]
```diff
@@ -324,10 +324,10 @@ <h2>Okta System Log内の<span class="mc-variable okta-feature-names.ScaleFT var
...
+                                                            <p>レポートをフィルタリングする日付範囲を指定します。<span class="mc-variable okta-variables.ProductName variable">Okta</span>はイベントを90日間保持するため、最大3か月前までさかのぼって範囲を指定できます。 </p>
                                                         </li>
                                                         <li value="3">
-                                                            <p>検索フィルターを使って監査イベントを検索します。たとえば、「<span class="userinput">(PAM)</span>」を検索すると、すべての<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>監査イベントが一覧表示されます。詳細については、「System Logのフィルターと検索」を参照してください。</p>
+                                                            <p>検索フィルターを使って監査イベントを検索します。たとえば、「<span class="userinput">(PAM)</span>」を検索すると、すべての<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>監査イベントが一覧表示されます。詳細については、「<a href="https://help.okta.com/oie/ja-jp/Content/Topics/Reports/syslog-filters.htm" target="_blank">System Logのフィルターと検索</a>」を参照してください。</p>
```
Comment (paulwallace-okta) [inline asa/ja-jp/content/topics/adv_server_access/docs/audit-events-with-okta-syslog.htm:330]: We have an absolute URL here. Making a note, will need to look at source.

File: asa/ja-jp/content/topics/adv_server_access/docs/gateway-configure.htm [hunk 3]
```diff
@@ -593,8 +593,21 @@ <h3><a name="capture"></a>セッションキャプチャのオプション</h3>
...
+                                                                            <div class="noteOkta">
+                                                                                <p class="noteContent">ゲートウェイは<code>/home</code>、<code>/root</code>、<code>/run/user</code>の各ディレクトリには書き込めません。</p>
+                                                                            </div>
+                                                                            <h5>例 - AWS</h5>
+                                                                            <p class="noteContent code">LogDestinations:<br />  - Type: s3<br />    Bucket: BUCKET-NAME<br />    Region: US-EAST-1<br />    # Use of ec2 instance IAM Role credentials for s3 bucket access is recommended<br />    # To use ec2 instance IAM Role credentials leave Profile, AccesKeyId, SecretKey, and SessionToken BLANK<br />    # Specify Profile for shared credentials<br />    Profile: AWS-PROFILE-NAME<br />    # Specify AccessKeyId, SecretKey, and SessionToken for static credentials<br />    AccessKeyId: SECRET<br />    SecretKey: SECRET<br />    SessionToken: SECRET</p>
```
Comment (paulwallace-okta) [inline asa/ja-jp/content/topics/adv_server_access/docs/gateway-configure.htm:607]: Making a note, will need to look at source. This is an odd way to format a code block.

**Review by emikhasyak-okta — APPROVED** [review]
LGTM! 🚢

**Pull request merged**